dockerrotate/main.py

```diff
@@ -7,6 +7,7 @@
 from dateutil import parser
 from docker import Client
 from docker.errors import APIError
+from docker.errors import NotFound
```

Comment:
This class does not exist in docker-py 1.1.0 which will be installed as a minimum version by setup.py. The class definitely exists by 1.6.0